docs/documentation/send-kin-ios.md

```diff
@@ -0,0 +1,109 @@
+---
+id: send-kin-ios
+title: Send Kin Module for iOS
+---
+
+This module is an optional way to allow your users sending and receiving Kin from their wallets in other apps, on the same device.
```

Comment:
The language here is a bit funny. Maybe instead something like, "This module is an optional way to allow your users to send and receive Kin from their wallets across different apps on the same device."